src/worker/tasks/nonceResyncWorker.ts

```diff
@@ -40,7 +40,7 @@ export const initNonceResyncWorker = async () => {
  * This is to unblock a wallet that has been stuck due to one or more skipped nonces.
  */
 const handler: Processor<any, void, string> = async (job: Job<string>) => {
-  const sentNoncesKeys = await redis.keys("nonce-sent*");
+  const sentNoncesKeys = await redis.keys("nonce-sent:*");
```

Comment:
typo but doesn't actually change behavior much since the previous command should have worked too.